### DISCUSSION TITLE
Use remix hooks + forms outside of remix?

### DISCUSSION THREAD

**Discussion by vedantroy** [remix-run/remix#3317] [Q&A] [answered]
Would it be possible to use a Remix `Form`+ `useLoaderData` outside of remix.run?
For context, I'm developing a Chrome extension, and I like the Remix dev experience. So I'd like to utilize remix's tools outside of remix.

**Comment by sergiodxa** [comment, marked as answer]
The v6.4.0 of React Router (in beta right now but can be installed as `react-router-dom@next`) has Form and useLoaderData.
